templates/rhdm70-kieserver.yaml

```diff
@@ -15,141 +15,141 @@ message: A new Decision Manager Execution Server application has been created in
   project. Please be sure to create the "kieserver-service-account" service account and
   the secret named "${HTTPS_SECRET}" containing the ${HTTPS_KEYSTORE} file used for
   serving secure content.
+  Only stateless API calls to the Decision Server are supported.
 parameters:
 - displayName: Application Name
   description: The name for the application.
   name: APPLICATION_NAME
   value: myapp
   required: true
 - displayName: EAP Admin User
-  description: EAP administrator username
+  description: EAP administrator user name. Use this user account if you need to log on to the JBoss EAP console.
```

Comment:
console is disabled by default, wouldn't better change to CLI and specify that user can execute rsh on the pod to access the CLi?